src/main/resources/assets/terrestria/lang/fr_fr.json

```diff
@@ -0,0 +1,283 @@
+{
+  "itemGroup.terrestria.items": "Terrestria",
+  "item.terrestria.cypress_boat": "Bateau en cyprès",
+  "item.terrestria.hemlock_boat": "Bateau en tsuga",
+  "item.terrestria.japanese_maple_boat": "Bateau en érable palmé",
+  "item.terrestria.rainbow_eucalyptus_boat": "Bateau en eucalyptus arc-en-ciel",
+  "item.terrestria.redwood_boat": "Bateau en séquoia",
+  "item.terrestria.rubber_boat": "Bateau en hévéa",
+  "item.terrestria.sakura_boat": "Bateau en sakura",
+  "item.terrestria.willow_boat": "Bateau en saule",
+  "block.terrestria.redwood_log": "Bûche de séquoia",
+  "block.terrestria.redwood_wood": "Bois de séquoia",
+  "block.terrestria.redwood_quarter_log": "Bûche de séquoia",
+  "block.terrestria.redwood_sapling": "Pousse de séquoia",
+  "block.terrestria.redwood_leaves": "Feuilles de séquoia",
+  "block.terrestria.redwood_planks": "Planches de séquoia",
+  "block.terrestria.redwood_slab": "Dalle en séquoia",
+  "block.terrestria.redwood_stairs": "Escalier en séquoia",
+  "block.terrestria.redwood_fence": "Barrière en séquoia",
+  "block.terrestria.redwood_fence_gate": "Portillon en séquoia",
+  "block.terrestria.redwood_door": "Porte en séquoia",
+  "block.terrestria.redwood_pressure_plate": "Plaque de pression en séquoia",
+  "block.terrestria.redwood_button": "Bouton en séquoia",
+  "block.terrestria.redwood_sign": "Pancarte en séquoia",
+  "block.terrestria.redwood_trapdoor": "Trappe en séquoia",
+  "block.terrestria.stripped_redwood_wood": "Bois de séquoia écorcé",
```

Comment:
This is not correct, as are most of the stripped translations. Bûche de séquoia écorcée is great, it translates to debarked, but most others translate to barked